be/src/cloud/cloud_meta_mgr.cpp

```diff
@@ -385,6 +385,10 @@ Status retry_rpc(std::string_view op_name, const Request& req, Response* res,
         } else if (res->status().code() == MetaServiceCode::INVALID_ARGUMENT) {
             return Status::Error<ErrorCode::INVALID_ARGUMENT, false>("failed to {}: {}", op_name,
                                                                      res->status().msg());
+        } else if (res->status().code() ==
+                   MetaServiceCode::KV_TXN_CONFLICT_RETRY_EXCEEDED_MAX_TIMES) {
+            return Status::Error<ErrorCode::DELETE_BITMAP_LOCK_ERROR, false>(
```

Comment:
Why return such error for all rpc? DELETE_BITMAP_LOCK_ERROR is only used for delete 
bitmap lock related rpc

Reply:
already remove and use another way to judge @zhannngchen